### DISCUSSION TITLE
Components accept any props, even non existing ones.

### DISCUSSION THREAD

**Discussion by bogdan2510** [wix/react-native-ui-lib#2190] [Q&A]
None of the components have type safety.  Ex: `<Text blabla  />` will not complain or give an error, even though this is  a non existing prop. Is this intentional? 

also i have done something like this :

```
Typography.loadTypographies({
testH1: {
    fontSize: themeFontSizes.big,
    fontWeight: '600',
    lineHeight: 45,
    fontFamily: themeFonts.semiBold,
  },
})
```

I expected to have intellisense for this when using it.  ex: `<Text testH1  />` works but no intellisense.

**Comment by hanayashiki** [comment]
Same case here. But when I get into the definition of  `Text`, it does have typing in props. I'm using TypeScript `4.8.3`, and using `strict` mode.

Any ideas there?

<img width="575" alt="image" src="https://user-images.githubusercontent.com/26056783/189652220-ae711253-a763-4b49-9c5b-9cbaa079e2c8.png">

**Reply by rajgopalv** [reply]
Did you all find any workaround for this ?

**Reply by bogdan2510** [reply]
Nope
